File: Data Structures & Algorithms/max-water-container/submission-0.java
class Solution {
    public int maxArea(int[] h) {
        int l=0;
        int r=h.length-1;
        int ma=0;
        while(l<r){
            int ca=Math.min(h[l],h[r])*(r-l);
            ma=Math.max(ma,ca);
            if(h[l]<h[r]){
                l++;
            }else{
                r--;
            }
        }
        return ma;
    }
}

File: Data Structures & Algorithms/maximum-number-of-balloons/submission-0.py
from collections import Counter
class Solution:
    def maxNumberOfBalloons(self, text: str) -> int:
        count=Counter(text)
        b=Counter("balloon")
        r=len(text)
        for c in b:
            r=min(r,count[c]//b[c])
        return r